versions.sh

```diff
@@ -171,22 +159,12 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
-	# TODO wheelVersion, somehow: https://github.com/docker-library/python/issues/365#issuecomment-914669320
-
-	echo "$version: $fullVersion (pip $pipVersion${setuptoolsVersion:+, setuptools $setuptoolsVersion}${hasWindows:+, windows})"
+	echo "$version: $fullVersion (${setuptoolsVersion:+setuptools $setuptoolsVersion}${hasWindows:+${setuptoolsVersion:+, }windows})"
```

Comment:
Hmm, this means 3.12+ will output `3.12: 3.12.XX ()` eventually once it's at the point where the Windows builds drop, but I think that's probably OK (I don't think there's a significantly cleaner way to write this that isn't going to be overkill just for this one edge case that's going to be a while before we hit :smile:).

Reply:
Ah good point :-)

What do you think about this?

```bash
	if [[ -n "${setuptoolsVersion}" || -n "${hasWindows}" ]]; then
		echo "$version: $fullVersion (${setuptoolsVersion:+setuptools $setuptoolsVersion}${hasWindows:+${setuptoolsVersion:+, }windows})"
	else
		echo "$version: $fullVersion"
	fi
```

Reply:
I'm thinking maybe the other extreme and just drop the extra data -- the `setuptools` versions won't be updated in a way that isn't tied to the Python version, and the `windows` is just a debugging tool to make sure the Windows versions get picked up correctly which is honestly something we'll only notice because they drop unexpectedly at which point we'll be explicitly debugging _that_ and probably using `bash -x` to do so, so it's really not providing much value anymore. :thinking:

_So_, how about this instead, and we Keep It Simple? :smile:

```suggestion
	echo "$version: $fullVersion"
```